app/scripts/plugin-loader.js

```diff
@@ -0,0 +1,18 @@
+import { Registry } from '@spinnaker/core';
+
+function loadPluginScript(plugin) {
+  return new Promise((resolve, reject) => {
+    var scriptTag = document.createElement('script');
+    scriptTag.src = plugin.location;
+    scriptTag.onload = () => resolve();
+    scriptTag.onreadystatechange = () => resolve();
+    scriptTag.onerror = () => reject();
+    document.body.appendChild(scriptTag);
+  });
+}
+
+export function initPlugins() {
```

Comment:
can you leave a comment on how this works?

Reply:
talked with @bpowell about this, for our notes:

- want less about how this is working, eg don't need to know that it calls the function above it to do stuff, but more around how this is part of the larger project. 
- talk about how its the entry point for plugins